plugins/inventory_sync/web/index.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useContext, useEffect, useState } from "react";
+
+import libPlugin from "@clusterio/lib/plugin";
+import { PageLayout, ControlContext } from "@clusterio/web_ui";
+import info from "../info";
+
+import "./index.css";
+
+function InventoryPage() {
+	let control = useContext(ControlContext);
+
+	let [statsData, updateStatsData] = useState();
+
+	useEffect(() => {
+		(async () => {
+			// Get statistics
+			updateStatsData(await info.messages.databaseStats.send(control, {}));
+		})();
+	}, []);
+
+	return <PageLayout nav={[{ name: "Inventory sync" }]}>
+		<h2>Inventory sync</h2>
+		{statsData && <>
+			<p>Database size: {Math.round(statsData.database_size / 1000)}kB</p>
+			<p>Database entries: {statsData.database_entries}</p>
+			<p>Largest entry is {statsData.largest_entry.name} with {(statsData.largest_entry.size / 1000)}kB</p>
+		</>}
```

Comment:
@Danielv123 any update on this comment?

Reply:
Master GUI hasn't been a high priority, I just wanted an easy way to display these select stats because we noticed it was important in identifying accidental bandwidth hogs during the gridlock event. Later on I expect it will include some sort of way to view and/or manage the players inventory and precense on various instances.

Reply:
Understood, maybe a good idea to make an issue for this and update it in a separate PR then?